### PR TITLE
[teleoperation_iCub_model_V_3] Uptdate the shoulder_pitch upper limit

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This updates the limits accordingly to robotology/robots-configuration@51b36db82171e5bf377170cb7c0802ef6e2c30c9

cc @S-Dafarra

### CODE REVIEW

File: conf/urdfs/teleoperation_iCub_model_V_3.urdf [hunk 1]
```diff
@@ -423,7 +423,7 @@
     <axis xyz="3.323079797673039e-07 0.9999999783534342 1.4452224303407313e-07"/>
     <parent link="chest"/>
     <child link="l_shoulder_1"/>
-    <limit effort="50000" lower="-3.14" upper="0.43" velocity="120.0"/>
+    <limit effort="50000" lower="-3.14" upper="0.34" velocity="120.0"/>
```
Comment (lrapetti) [inline conf/urdfs/teleoperation_iCub_model_V_3.urdf:426]:
```suggestion
    <limit effort="50000" lower="-3.05" upper="0.34" velocity="120.0"/>
```
Lower limit seems to be also wrong

File: conf/urdfs/teleoperation_iCub_model_V_3.urdf [hunk 2]
```diff
@@ -661,7 +661,7 @@
     <axis xyz="1.499044390054707e-07 0.9999999783534812 -1.2401537036188515e-07"/>
     <parent link="chest"/>
     <child link="r_shoulder_1"/>
-    <limit effort="50000" lower="-3.14" upper="0.43" velocity="120.0"/>
+    <limit effort="50000" lower="-3.14" upper="0.34" velocity="120.0"/>
```
Comment (lrapetti) [inline conf/urdfs/teleoperation_iCub_model_V_3.urdf:664]:
```suggestion
    <limit effort="50000" lower="-3.05" upper="0.34" velocity="120.0"/>
```

**Comment by GiulioRomualdi** [comment]
@lrapetti I applied your changes merge it whenever you want :)

**Comment by lrapetti** [comment]
> @lrapetti I applied your changes merge it whenever you want :)

Thanks!

**Pull request merged**